src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,55 @@
+@import "../../../../public/variables";
+
+$firstBreadcrumbPolygon: polygon(
+  0% 0%,
+  100% 0%,
+  calc(100% - #{$euiSizeS}) 100%,
+  0% 100%
+);
+$breadcrumbPolygon: polygon(
+  $euiSizeS 0%,
+  100% 0%,
+  calc(100% - #{$euiSizeS}) 100%,
+  0% 100%
+);
+
+.osdHeaderBreadcrumbs {
+  /*
+  filter defines a custom filter effect by grouping atomic filter primitives!
+  here we are using Gaussian filter with stdDeviation for applying
+  border-radius on clipped background.
+  */
+
+  filter: url("../../public/assets/round_filter.svg#round");
+
+  .osdBreadcrumbs {
+    clip-path: $breadcrumbPolygon;
+    background-color: $osdHeaderBreadcrumbGrayBackground;
+    padding: $euiSizeXS - 2.5 $euiSizeL - $euiSizeXS;
+
+    &:first-child {
+      clip-path: $firstBreadcrumbPolygon;
+    }
+
+    &:last-child {
+      background-color: $osdHeaderBreadcrumbBlueBackground;
+    }
+  }
+
+  .euiBreadcrumbSeparator {
+    display: none;
+  }
+
+  .euiBreadcrumb__collapsedLink {
+    color: $osdHeaderBreadcrumbCollapsedLink;
+    background: $euiColorEmptyShade;
+  }
+
+  .euiPopover__anchor {
+    padding: 0 $euiSizeS;
+  }
+
+  .euiBreadcrumb:not(.euiBreadcrumb:last-child) {
```

Comment:
`.euiBreadcrumb:not(.euiBreadcrumb:last-child) {` can simply be `.euiBreadcrumb:not(:last-child) {`

Reply:
just overriding the already existing css! so going with what they already have!